packages/table/preview/index.tsx

```diff
@@ -941,3 +953,13 @@ function getRandomInteger(min: number, max: number) {
 function contains(arr: any[], value: any) {
     return arr.indexOf(value) >= 0;
 }
+
+function localStorageGetOrDefault(stateKey: keyof IMutableTableState, defaultValue: any) {
```

Comment:
Did you name it this to mimic `localStorage.get...`? Either way, I'd prefer something with a verb in front. Maybe `getLocalStorageValueOrDefault`. Don't feel strongly though.